docs/source/tutorials/running_a_simulation.rst

```diff
@@ -0,0 +1,145 @@
+=====================
+Running a Simulation
+=====================
+
+Once you've built a model specification that describes the plugins, components, and configuration for the model you
+want to run, it's time to run the simulation defined by that model specification using ``vivarium``. There are two ways
+to run a single simulation. We will describe first the process for running a simulation from the command line.
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+Running from the Command Line
+------------------------------
+To run from the command line, we'll use the :func:`vivarium.interface.cli.simulate` command. This command is actually
+a group containing three subcommands: ``run``, ``test``, and ``profile``. We will focus on ``run`` here. We'll start
+by covering how to use this command to run a simulation from a model specification and then cover at a high-level what
+happens when your model is running.
+
+
+How to Use ``simulate run``
++++++++++++++++++++++++++++
+The basic use of ``simulate run`` requires no more than a model specification yaml file. With this, you can do the
+following to run the model defined by that specification:
+
+.. code-block:: console
+
+    simulate run /path/to/your/model/specification
+
+By default, ``simulate run`` will use ~/vivarium_results as the results directory. If you navigate to that directory,
+you should see a subdirectory with the name of your model specification has been created, within which is another
+subdirectory named for the start time of the run. In here, you should see two hdf files: final_state.hdf, which is the
+population state table at the end of the simulation, and output.hdf, which is the results of the metrics generated by
+the simulation.
+
+``simulate run`` also provides various flags which you can use to configure options for the run. These are:
+
+.. list-table:: **Available** ``simulate run`` **options**
+    :header-rows: 1
+    :widths: 30, 40
+
+    *   - Option
+        - Description
+    *   - | **--results-directory** or **-o**
```

Comment:
you should add --help maybe, so they know they can access this without going all the way to the docs